python-threatexchange/threatexchange/api.py

```diff
@@ -544,7 +591,7 @@ def get_threat_descriptors_from_indicator(
             self._base_url
             + "/"
             + str(indicator_id)
-            + "?fields=descriptors{privacy_members,indicator,type,owner}&access_token="
```

Comment:
This fails if any descriptor returned is owned by a different app

Reply:
blocking, not sure I understand this change don't we just want to remove owner?

Reply:
sorry, `privacy_members` leads to failure if the owner ID is not our app. Could just remove privacy members but I think it might be better to return all default fields not just indicator, type, and owner

Reply:
oh, this is undoing your change in #584, missed that. good to go